tests/testapps/AddressBookSample/AddressBookSample.vsimporter/AddressBookSample-WinStore10/Package.appxmanifest

```diff
@@ -1,49 +1,29 @@
-<?xml version="1.0" encoding="utf-8"?>
-
-<Package
-  xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10"
-  xmlns:mp="http://schemas.microsoft.com/appx/2014/phone/manifest"
-  xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10"
-  IgnorableNamespaces="uap mp">
-
-  <Identity
-    Name="34906fc5-08c8-4bad-b833-46a91bb7cf95"
-    Publisher="islandwood"
-    Version="1.0.0.0" />
-
-  <mp:PhoneIdentity PhoneProductId="34906fc5-08c8-4bad-b833-46a91bb7cf95" PhonePublisherId="00000000-0000-0000-0000-000000000000"/>
-
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Package xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10" xmlns:mp="http://schemas.microsoft.com/appx/2014/phone/manifest" xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10" IgnorableNamespaces="uap mp">
+  <Identity Name="34906fc5-08c8-4bad-b833-46a91bb7cf95" Publisher="islandwood" Version="1.0.0.0" />
+  <mp:PhoneIdentity PhoneProductId="34906fc5-08c8-4bad-b833-46a91bb7cf95" PhonePublisherId="00000000-0000-0000-0000-000000000000" />
   <Properties>
     <DisplayName>AddressBookSample</DisplayName>
     <PublisherDisplayName>t-chmou</PublisherDisplayName>
     <Logo>Assets\StoreLogo.png</Logo>
   </Properties>
-
   <Dependencies>
     <TargetDeviceFamily Name="Windows.Universal" MinVersion="10.0.0.0" MaxVersionTested="10.0.0.0" />
   </Dependencies>
-
   <Resources>
     <Resource Language="x-generate" />
   </Resources>
-
   <Applications>
-    <Application Id="App"
-      Executable="$targetnametoken$.exe"
-      EntryPoint="AddressBookSample.App">
-      <uap:VisualElements
-        DisplayName="AddressBookSample"
-        Square150x150Logo="Assets\Square150x150Logo.png"
-        Square44x44Logo="Assets\Square44x44Logo.png"
-        Description="AddressBookSample"
-        BackgroundColor="transparent">
-        <uap:DefaultTile Wide310x150Logo="Assets\Wide310x150Logo.png"/>
+    <Application Id="App" Executable="$targetnametoken$.exe" EntryPoint="AddressBookSample.App">
+      <uap:VisualElements DisplayName="AddressBookSample" Square150x150Logo="Assets\Square150x150Logo.png" Square44x44Logo="Assets\Square44x44Logo.png" Description="AddressBookSample" BackgroundColor="transparent">
+        <uap:DefaultTile Wide310x150Logo="Assets\Wide310x150Logo.png">
+        </uap:DefaultTile>
         <uap:SplashScreen Image="Assets\SplashScreen.png" />
       </uap:VisualElements>
     </Application>
   </Applications>
-
   <Capabilities>
     <Capability Name="internetClient" />
+    <uap:Capability Name="contacts" />
```

Comment:
><uap:Capability Name="contacts" /> [](start = 4, length = 34)

Do we need a vsimporter change to insert this into app manifests if they're using the AddressBook framework?